exchanges/oex/oex_wrapper.go

```diff
@@ -0,0 +1,384 @@
+package oex
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// Start starts the Oex go routine
+func (o *Oex) Start(wg *sync.WaitGroup) {
+	wg.Add(1)
+	go func() {
+		o.Run()
+		wg.Done()
+	}()
+}
+
+// Run implements the Oex wrapper
+func (o *Oex) Run() {
+	if o.Verbose {
+		log.Debugf("%s polling delay: %ds.\n", o.GetName(), o.RESTPollingDelay)
+		log.Debugf("%s %d currencies enabled: %s.\n", o.GetName(), len(o.EnabledPairs), o.EnabledPairs)
+	}
+	exchangeCurrencies, err := o.GetAllPairs()
+	if err != nil {
+		log.Errorf("%s Failed to get available symbols.\n", o.GetName())
+	}
+	var newExchangeCurrencies currency.Pairs
+	for x := 0; x < len(exchangeCurrencies.Data); x++ {
+		newExchangeCurrencies = append(newExchangeCurrencies, currency.NewPairFromString(exchangeCurrencies.Data[x].Symbol))
```

Comment:
As per the other message about adjusting the delimiter, please construct the pair so that you use it here

Reply:
Can you pls double check this one to make sure its done correctly since I was a bit confused about this one.